src/platform/unittest/TlsTest.cpp

```diff
@@ -1818,4 +1865,181 @@ TEST_F(TlsTest, CipherSuiteInvalid)
     }
 }
 
+_Function_class_(CXPLAT_SEC_CONFIG_CREATE_COMPLETE)
+static void
+QUIC_API
+SchannelSecConfigCreateComplete(
+    _In_ const QUIC_CREDENTIAL_CONFIG* /* CredConfig */,
+    _In_opt_ void* Context,
+    _In_ QUIC_STATUS Status,
+    _In_opt_ CXPLAT_SEC_CONFIG* SecConfig
+    )
+{
+#if QUIC_TLS_SCHANNEL
```

Comment:
I see why you are making these defined, but I wish there was a way to do it without these. I've tried very hard up until now to only make feature level flags to control the testing of different behavior, like `QUIC_DISABLE_0RTT_TESTS` or `QUIC_DISABLE_DEFERRED_CERT_TESTS`. Could we instead make a flag like that for this?

Reply:
I can make that change, but it feels like a wasted effort since the point is that the test should pass for one TLS implementation, and fail on the other TLS implementation. The feature flag would depend on the TLS implementation that's active.